Author: Karsten Koop
Description: bootstrap 3 breaks bootstrap-tour
 A recent security update CVE-2019-8331 breaks bootstrap tour
 because the sanitation removes the next and prev buttons from
 the popover.
Forwarded: https://github.com/sorich87/bootstrap-tour/issues/729

--- node-bootstrap-tour.orig/src/coffee/bootstrap-tour.coffee
+++ node-bootstrap-tour/src/coffee/bootstrap-tour.coffee
@@ -517,6 +517,7 @@
         title: step.title
         content: step.content
         html: true
+        sanitize: false
         animation: step.animation
         container: step.container
         template: step.template
